ddp-workspace/projects/ddp-sdk/src/lib/models/statistic.ts

```diff
@@ -0,0 +1,14 @@
+export interface Statistic {
+  configuration: {
+    type: string;
```

Comment:
Probably it would be better to create an enum with acceptable values: `PARTICIPANTS` `DISTRIBUTION` `MAILING_LIST` `SPECIFIC_ANSWER` `KITS` instead of string type.

Reply:
Fixed. This type fully depends on the study configuration so this has to be generic.